evm/deploy-its.js

```diff
@@ -197,20 +162,20 @@ async function deployImplementation(wallet, chain, deploymentKey, skipExisting =
         implementation: {
             name: 'Interchain Token Service Implementation',
             async deploy() {
-                return await deployCreate2(
-                    create2Deployer,
+                return await deployContract(
+                    deployOptions.deployMethod,
                     wallet,
                     InterchainTokenService,
                     [
                         contractConfig.tokenManagerDeployer,
                         contractConfig.standardizedTokenDeployer,
                         contracts.AxelarGateway.address,
                         contracts.AxelarGasService.address,
-                        contractConfig.remoteAddressValidator,
+                        contractConfig.interchainTokenFactory,
+                        chain.name,
```

Comment:
```suggestion
                        chain.id,
```
I'd like to use chain id here to guarantee uniqueness